src/index.html

```diff
@@ -2,15 +2,56 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      class="form first" 
+      action="#" 
+      method="get" 
+      data-qa="big"
+    >
+
+      <label for="bigSearch" class="label" >
+        
+        <img src= "images/Search.svg" alt="search" class="images_large">
+
+        <input 
+          type="text" 
+          class="input_large" 
+          id="bigSearch" 
+          data-qa="keypress"  
+          placeholder="Try &quot;Los Angeles&quot;"
+        >
+
+      </label>
+
+    </form>
+
+    <form 
+      class="form second" 
+      action="#" 
+      method="get" 
+      data-qa="small"
+    >
+
+      <label for="smallSearch" class="label2" tabindex="0">
+
+        <img src= "images/Search.svg" alt="search" class="images_small">
```

Comment:
remember to put attributes on new lines if there are 3 and more of them
```suggestion
        <img
          src= "images/Search.svg"
          alt="search"
          class="images_small"
        >
```